CONTRIBUTING.md

```diff
@@ -147,6 +147,14 @@ Here's some examples of what we're trying to avoid:
 - Adds a feature
 - Feature now does something
 
+### Downloading Pulumi from contributed pull requests
+
+Artifacts built during pull request workflows can be downloaded by running the following command:
+
+```sh
+curl -fsSL https://get.pulumi.com | sh -s -- --version pr#NUMBER
```

Comment:
I was initially confused whether the `#` is necessary… wondering if showing it this way would be more clear?

```suggestion
curl -fsSL https://get.pulumi.com | sh -s -- --version pr#<number>
```

Reply:
oh yeh that's better